### ISSUE TITLE
[Feature Request] DIfferent download locations for playlists/Download location in the homepage

### ISSUE THREAD

**Issue by Nikko-A** [aandrew-me/ytDownloader#78]
Greetings! 

I have a recurring playlist that I frequently update and re-download with videos I use as resources for editing (thank god this program skips already-downloaded files, although slowly). I re-organized this playlist into a different folder compared to the singular video downloads and with their own respective tags. I was wondering if it was possible to allow a different download location for playlists compared to singular videos. 

Maybe this isn't a good solution and may complicate the code of this program just to accommodate my workflow, so I'd like to ask if it would be possible to instead move the "change download location" into the homepage/front screen so as to hasten this process of organization. 

I hope I'm not imposing much since I truly do love this program, it's an amazing utility and I haven't been able to find a different utility like it for Linux. Much appreciated to the work you do!

Edit: Wanted to delete this issue but couldn't. Apologies for that.

**Comment by aandrew-me** [comment]
Hey man!

Don't hesitate to make requests. After all this program exists to make people's lives easier.
Adding support for a separate default download location for playlists is no big deal. 

Moreover having the option to change download location in home page (under **More Options**) also sounds like a good idea.

**Comment by Nikko-A** [comment]
I really appreciate the kindness!

I will admit, I closed this suggestion down mostly because it felt like the solution completely solve the issue I had the more I thought about it. 

Although making the "change download" prompt more visible is helpful, the separate download locations for playlist was an idea I pitched as a bi-product of a separate issue. I needed the tagging system to be consistent across several videos. Since there was no way to change the names of the playlist download, I thought it would be enough if the playlist would just stay in a specific spot and would instead be constantly checked. This created several other issues I thought so I decided to scrap that suggestion and created new ones that would through #80 and #81 

I felt kind of, for lack of a better word, shy pitching an idea that I think would only benefit me, so I chose to suggest different ideas that might solve any forseeable issues in regards to naming and the like. 

I actually wrote down the issues and tried to devise a plan like this:
![image](https://user-images.githubusercontent.com/74495189/202911691-cc859091-40e7-411f-a5bc-ed463b958bd0.png)
